packages/cloudflare/src/cli/index.ts

```diff
@@ -1,63 +1,25 @@
 #!/usr/bin/env node
-import { createRequire } from "node:module";
-import path from "node:path";
 
-import { compileOpenNextConfig } from "@opennextjs/aws/build/compileConfig.js";
-import { normalizeOptions } from "@opennextjs/aws/build/helper.js";
-import { printHeader, showWarningOnWindows } from "@opennextjs/aws/build/utils.js";
-import logger from "@opennextjs/aws/logger.js";
-import { unstable_readConfig } from "wrangler";
+import yargs from "yargs";
 
-import { Arguments, getArgs } from "./args.js";
-import { build } from "./build/build.js";
-import { createOpenNextConfigIfNotExistent, ensureCloudflareConfig } from "./build/utils/index.js";
-import { deploy } from "./commands/deploy.js";
-import { populateCache } from "./commands/populate-cache.js";
-import { preview } from "./commands/preview.js";
-import { upload } from "./commands/upload.js";
-import { getWranglerConfigFlag, getWranglerEnvironmentFlag } from "./utils/run-wrangler.js";
+import { addBuildCommand } from "./commands/build.js";
+import { addDeployCommand } from "./commands/deploy.js";
+import { addPopulateCacheCommand } from "./commands/populate-cache.js";
+import { addPreviewCommand } from "./commands/preview.js";
+import { addUploadCommand } from "./commands/upload.js";
 
-const nextAppDir = process.cwd();
+export function runCommand() {
+	let y = yargs(process.argv.slice(2))
+		.scriptName("opennextjs-cloudflare")
+		.parserConfiguration({ "unknown-options-as-args": true });
 
-async function runCommand(args: Arguments) {
-	printHeader(`Cloudflare ${args.command}`);
+	y = addBuildCommand(y);
```

Comment:
maybe this can return y for chaining?
Up to you

Reply:
Not sure I follow - return it where?

Reply:
Nevermind... 
Do you need the y= or would the function mutate y?